redis_enterprise - enforce metric types (ECOINT-132) (#2617)
DataDog/integrations-extras: redis_enterprise/README.md

Set the `openmetrics_endpoint` to point to your cluster. See the [example][4]. Leave `tls_verify` set to false.

> If you have more than one endpoint it will be necessary to create additional instances with the respective URLs.

Reviewer: ```suggestion
If you have more than one endpoint, it is necessary to create additional instances with the respective URLs.
```

> An example is given below, and is also noted in its own [example][5]:

Reviewer: ```suggestion
See the example below, which is also in [conf.yaml.multi.example][5]:
```
